cocos/animation/marionette/state-machine/condition/binding/binding.ts

```diff
@@ -0,0 +1,46 @@
+import { ConditionEvalContext } from '../condition-base';
+
+/**
+ * @zh
+ * 描述过渡条件中，某个绑定提供的值的类型。
+ * @en
+ * Describes the type of value providing by a transition condition binding.
+ */
+export enum TCBindingValueType {
+    FLOAT = 0,
+
```

Comment:
Is there a place holder?

Reply:
By design, I wish the enumerators in `TCBindingValueType` to have number value equal to corresponding enumerator in `VariableType`. Due to typescript limit it's not currently possible to do:

```ts
enum TCBindingValueType { FLOAT = VariableType.FLOAT };

interface Map {
  [TCBindingValueType.FLOAT]: number; // Error here! `TCBindingValueType.FLOAT` can not be used as a property signature key!
}
```

So I have to duplicate its actual value.

However, I've document it on the side. See: https://github.com/cocos/cocos-engine/pull/14857/commits/979f0ffae8f77f6847189ae6254f1b49f062b08f#diff-e123ce10ac6a31e8c7cdb074c055046baac7942d98b98bb9e8eabd4b643bad41L9